script/openqa-verify-pr

```diff
@@ -0,0 +1,119 @@
+#
+# Parameters:
+#  SOURCE - The source you want to schedule a run for. Can be either 'opensuse' or 'suse'.
+#  PR_ID - The ID of your GitHub pull request.
+#  BASE_TEST_ID - The ID of the test you want you use as a base.
+#  [test/module,...] - The modules of your test you want to schedule for.
+#  TEST_NAME - The Name of your test. (E.g.: TEST=RUST)
+#
+# Specify your GitHub access token in '~/.config/openqa/verification.conf' like this:
+#
+# GH_ACCESS_TOKEN="YOUR_TOKEN_HERE"
+#
+# To specify the URL to your own openQA instance specify it in the same config file like this:
+#
+# CUSTOM_TARGET="https://openqa.yourdomain.com"
+#
+# Default test modules
+#  'tests/installation/bootloader_start'
+#  'tests/boot/boot_to_desktop'
+#
+# Options:
+#  -h, --help        Display this help text.
+# 
+# Example
+#
+# Pass arguments to this script like this:
+#
+# schedule_verification_run opensuse TOKEN 1234 12345 tests/console/rustup,tests/console/cargo RUST
+
+
+help_text=$(cat <<'EOF'
+OpenQA verification run scheduler
+
+Usage:
+ openqa-verify-pr $SOURCE $PR_ID $BASE_TEST_ID $[test/module, test/module,...] $TEST_NAME
+
+Parameters:
+ SOURCE - The source you want to schedule a run for. Can be either 'opensuse' or 'suse'.
+ PR_ID - The ID of your GitHub pull request.
+ BASE_TEST_ID - The ID of the test you want you use as a base.
+ [test/module,...] - The modules of your test you want to schedule for.
+ TEST_NAME - The Name of your test. (E.g.: TEST=RUST)
+
+Specify your GitHub access token in '~/.config/openqa/verification.conf' like this:
+
+GH_ACCESS_TOKEN="YOUR_TOKEN_HERE"
+
+To specify the URL to your own openQA instance specify it in the same config file like this:
+
+CUSTOM_TARGET="https://openqa.yourdomain.com"
+
+Default test modules:
+ 'tests/installation/bootloader_start'
+ 'tests/boot/boot_to_desktop'
+
+Options:
+ -h, --help        Display this help text.
+
+EOF
+)
+
+set -o pipefail
+
+source=$1 pr_id=$2 test_id=$3 modules=$4 test_name=$5
+config_file=$HOME/.config/openqa/verification.conf config= custom_target=
+
+if [ $1 = '-h' ] || [ $1 = '--help' ]; then
+  echo "$help_text"
+  exit 0
+fi
+
+# Validate argument count
+if [ $# -ne 5 ]; then
+  echo "Invalid number of arguments. Run with '-h' or '--help' to view usage."
+  exit 1
+fi
+
+# Read GH token from config file.
+if [[ -f $config_file ]]; then
+  config=$(grep "^GH_ACCESS_TOKEN=" "$config_file" | cut -d'"' -f2)
+  custom_target=$(grep "^CUSTOM_TARGET=" "$config_file" | cut -d'"' -f2)
+else
+  echo "No config file found! Run with '-h' for more info."
+  exit 1
+fi
+
+export GITHUB_TOKEN=$config
+
+if [ $source = 'opensuse' ]; then
+  echo "Dispatching verification run for opensuse..."
+  openqa-clone-custom-git-refspec "https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/$pr_id" \
+    "https://openqa.opensuse.org/tests/$test_id" \
+    SCHEDULE="tests/installation/bootloader_start,tests/boot/boot_to_desktop,$modules" \
```

Comment:
Remember to move this to the  [openqa-clone-custom-git-refspec](https://github.com/os-autoinst/openQA/pull/5647/files#diff-1da53224de38473695839b4d4c7efee7a2add51a1aa9a3c259e872a5663b9498) behind a switch i.e `--modules a,b,c` so it gets appended via the for loop, before calling cone job.

and then there's the consideration that if the caller is passing:
- YAML_SCHEDULE + SCHEDULE, die (as one of them has to be overwritten.
- if the switch is passed, and so is either (yaml_schedule or schedule) die, as options aren't compatible

Reply:
Yep, currently looking into that :)